_posts/2017-11-02-audiophiler.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  Creating Audiophiler
+date:   2017-11-02 21:25:00
+categories: projects
+description: Audiophiler, a neat python web app for serving music
+author: Stephen Greene
```

Comment:
Would you be able to add a post image? I would make something similar to the mockups I made for the [CSH Gallery README](https://github.com/liam-middlebrook/gallery/blob/master/README.md), that don't show CSH member data, buts how how the site works.

Reply:
Agreed. Without images it's currently just going to be black, which is not going to look great.